kik_unofficial/client.py

```diff
@@ -44,20 +47,23 @@ def __init__(self, callback: callbacks.KikClientCallback, kik_username: str, kik
         :param kik_password: the kik password to log in with.
         :param kik_node: the username plus 3 letters after the "_" and before the "@" in the JID. If you know it,
                          authentication will happen faster and without a login. otherwise supply None.
-        :param device_id: a unique device ID. If you don't supply one, a random one will be generated. (generated at _on_connection_made)
+        :param device_id: a unique device ID. If you don't supply one, a random one will be generated. (generated in __init__ if None)
         :param android_id: a unique android ID. If you don't supply one, a random one will be generated.
-        :param enable_logging: If true, turns on logging to stdout (default: False)
+        ;param log_level: Level of logging 1 debug, 2 info, 3 warn, 4 error, 5 critical.
+        ;param logger_name; Name to use for the logger.
         ;param log_file_path: If set will create a daily rotated log file and archive for 7 days.
+
         """
-        # turn on logging with basic configuration
-        if enable_logging:
-            self.log = set_up_basic_logging(log_level=log_level, logger_name="kik_unofficial", log_file_path=log_file_path)
+
+        globals.LOGGER_NAME = logger_name
+
```

Comment:
I think it's a little weird that you set a global that appears to be constant here in the constructor. You can probably avoid it

Reply:
Its a variable from the globals.py file. Doing it this way allows other files in the project to use whatever variables in it as a global variable. 

I may be wrong, and it might be able to be avoided, but logging in other files has to be able to get the correct logger by name when set. 

`log = logging.getLogger(globals.LOGGER_NAME)`

Reply:
It's either global or per `KikClient`

Reply:
I see what you mean. I did not account for that. Is there another solution that might work?

Reply:
I don't think there is a need to unify the API logs and the bot logs. If you think it's needed, you can have `logger_name` as a member of `KikClient` and have a function to export that for whoever.

Reply:
I have removed the globals and setting the logger_name.